microsoft-edge/devtools-guide-chromium/whats-new/2024/10/devtools-130.md

```diff
@@ -0,0 +1,155 @@
+
+![Performance tab warning icon](./devtools-130-images/performance-warning-icon.png)
+
+See also:
+* [Throttle the network and CPU](../../../device-mode/index.md#throttle-the-network-and-cpu) in _Emulate mobile devices (Device Emulation)_.
+
+
+<!-- ====================================================================== -->
+## Console's Warnings count button has increased contrast
+
+<!-- Subtitle: The warning icon and expand button on warning messages in the Console now meet the standard luminosity ratio of 3:1.-->
+
+In the **Console** tool, the Warning count expander button on warning messages in the **Console** previously didn't meet the standard luminosity ratio of 3:1:
+
+![Low luminosity ratio for the Warning count expander button](./devtools-130-images/console-icon-luminosity-ratio-before.png)
+
+This has now been fixed; the Warning count expander button has a higher luminosity ratio:
+
+![High luminosity ratio for the Warning count expander button](./devtools-130-images/console-icon-luminosity-ratio-after.png)
+
+
+See also:
+* [Inspect and filter information on the current webpage](../../../console/index.md#inspect-and-filter-information-on-the-current-webpage) in _Console overview_.
+
+
+<!-- ====================================================================== -->
+## Announcements from the Chromium project
+
+Microsoft Edge 130 also includes the following updates from the Chromium project:
+
+* [Network panel improvements](https://developer.chrome.com/blog/new-in-devtools-130#network)
+   * [Network filters reimagined](https://developer.chrome.com/blog/new-in-devtools-130#network-filters)
+   * [HAR exports now exclude sensitive data by default](https://developer.chrome.com/blog/new-in-devtools-130#har)
+* [Elements panel improvements](https://developer.chrome.com/blog/new-in-devtools-130#elements)
+   * [Autocomplete values for text-emphasis-* properties](https://developer.chrome.com/blog/new-in-devtools-130#autocomplete)
+   * [Scroll overflows marked with a badge](https://developer.chrome.com/blog/new-in-devtools-130#scroll-badge)
+   * [Bare declarations after nested rules don't "shift up"](https://developer.chrome.com/blog/new-in-devtools-130#css-nesting)
+* [Performance panel improvements](https://developer.chrome.com/blog/new-in-devtools-130#perf)
+   * [Recommendations in live metrics](https://developer.chrome.com/blog/new-in-devtools-130#live-metrics-recommendations)
+   * [Navigate breadcrumbs](https://developer.chrome.com/blog/new-in-devtools-130#breadcrumbs)
+* [Memory panel improvements](https://developer.chrome.com/blog/new-in-devtools-130#memory)
+   * [New 'Detached elements' profile](https://developer.chrome.com/blog/new-in-devtools-130#detached-elements)
+   * [Improved naming of plain JS objects](https://developer.chrome.com/blog/new-in-devtools-130#object-names)
```

Comment:
Same thing here. I believe this is covered by the **Improved names of JavaScript objects in heap snapshots** section.

Reply:
removed chromium links:
```
* [Memory panel improvements](https://developer.chrome.com/blog/new-in-devtools-130#memory)
   * [New 'Detached elements' profile](https://developer.chrome.com/blog/new-in-devtools-130#detached-elements)
   * [Improved naming of plain JS objects](https://developer.chrome.com/blog/new-in-devtools-130#object-names)
```

* [Memory panel improvements](https://developer.chrome.com/blog/new-in-devtools-130#memory)
   * [New 'Detached elements' profile](https://developer.chrome.com/blog/new-in-devtools-130#detached-elements)
   * [Improved naming of plain JS objects](https://developer.chrome.com/blog/new-in-devtools-130#object-names)

@leahmsft